docs/running/tardis_widgets/generating_widgets.ipynb

```diff
@@ -4,18 +4,24 @@
    "cell_type": "markdown",
```

Comment:
<p>TARDIS, for consistency</p><p> </p><p>"as an HDF"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1280/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>